main.js

```diff
@@ -0,0 +1,286 @@
+    windSpeed = measures === 'imperial' ? 'миль/ч' : 'м/с'
+    searchInput.placeholder = 'Поиск города'
+    searchButton.innerText = 'поиск'
+    todayDescription.innerHTML = `<span>${description}</span>
+          <span>ощущается: ${Math.round(feels_like)}°</span>
+          <span>ветер: ${wind} ${windSpeed}</span>
+          <span>влажность: ${humidity}%</span>`
+      break
+    case 'en':
+    localStorage.setItem('language', 'en')
+    setDateAndTime()
+    mapLatitude.innerHTML = `<span>Latitude: ${mapLatitude.textContent}</span>`
+    mapLongitude.innerHTML = `<span>Longitude: ${mapLongitude.textContent}</span>`
+    windSpeed = measures === 'imperial' ? 'mph' : 'm/s'
+    searchInput.placeholder = 'Search city'
+    searchButton.innerText = 'search'
+    todayDescription.innerHTML = `<span>${description}</span>
+          <span>feels like: ${Math.round(feels_like)}°</span>
+          <span>wind: ${wind} ${windSpeed}</span>
+          <span>humidity: ${humidity}%</span>`
+      break
+  }
+}
+
+function setDateAndTime() {
+  const date = new Date();
+  const options = {
+    timeZone: timeZoneValue,
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    weekday: 'short',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    timeZoneName: 'short',
+  }
+  time.innerHTML = date.toLocaleString(`${languageOptions.value}`, options)
+  setTimeout(() => setDateAndTime(), 1000)
+  getDayOfWeek(date.getDay())
+}
+
+function getDayOfWeek(dayNumber) {
+  const weekDaysRu = ['Воскресенье', 'Понедельник', 'Вторник', 'Среда', 'Четверг', 'Пятница', 'Суббота'];
+  const weekDaysEn = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  let daysArray;
+  if(localStorage.getItem('language') === 'ru') {
+    daysArray = weekDaysRu;
+  } else {
+    daysArray = weekDaysEn;
+  }
+  for (let i = 0; i < severalDays.length; i += 1) {
+    dayNumber += 1;
+    if (dayNumber === 7) {
+      dayNumber = 0;
+    }
+    
+    severalDays[i].textContent = daysArray[dayNumber];
+  }
+}
+
+async function searchLocation() {
+  try {
+  searchCity = searchInput.value;
+  await getPositionByOpenCage()
+  searchInput.value = '';
+  } catch (e) {
+    getError(searchInput, `ERROR(${e.code}): ${e.message}`);
+  }
+}
+
+function setIcon(id) {
+  if (id<300 && id>=200) {
+    src = '../img/thunderstorms.svg';
+    return src;
+  } else if (id<400 && id>=300) {
+    src = '../img/drizzle.svg';
+    return src;
+  } else if (id<600 && id>=500) {
+    src = '../img/rain.svg';
+    return src;
+  } else if(id<700 && id>=600) {
+   src = '../img/snow.svg';
+   return src;
+  } else if(id<800 && id>700) {
+  src = '../img/mist.svg';
+  return src;
+  } else if (id==800){
+  src = '../img/clear-day.svg';
+  return src;
+  } else if (id<810 && id>800){
+  src = '../img/cloudy.svg';
+  return src;
+  } else  {
+  src = '../img/partly-cloudy-day.svg';
+  return src;
+  }
```

Comment:
You may `return '../img'` at once, there is no need in assigning it to the `src`.

Reply:
And I reckon that a switch case would be more readable here.